app/services/service_offering/add_to_portfolio_item.rb

```diff
@@ -16,8 +16,6 @@ def process
       end
 
       # Get the fields that we're going to pull over
-      create_param_map(determine_valid_fields)
-
       @item = PortfolioItem.create!(populate_missing_fields)
```

Comment:
I find `PortfolioItem.create!(populate_missing_fields)` confusing from a method naming standpoint.

Would it make more sense to call `create_param_map` here and that internally calls `populate_missing_fields`?